docs/admin/admission-controllers.md

```diff
@@ -256,6 +269,17 @@ applies a 0.1 CPU requirement to all Pods in the `default` namespace.
 
 See the [limitRange design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_limit_range.md) and the [example of Limit Range](/docs/tasks/configure-pod-container/limit-range/) for more details.
 
+### MutatingAdmissionWebhook (beta in 1.9)
+
+This plug-in calls any mutating webhooks which match the request. Matching
+webhooks are called in serial; each one may modify the object if it desires.
+
+This plugin (as implied by the name) only runs in the mutating phase.
+
+If a webhook called by this has side effects (e.g., decrementing quota) it
+*must* have a reconcilation system, as it is not guaranteed that subsequent
+webhooks or validating plugins will permit the request to finish.
+
```

Comment:
Add the following text:

Use caution when authoring and installing mutating webhooks.  
*  Users may be confused when the objects they try to create are different from what they get back. 
  * Setting originally unset fields is less confusing that overwriting fields set in the request.  Avoid doing the latter.  
* Built in control loops may break when the objects they try to create are different when read back.
  * Setting originally unset fields is less likely to cause problems than overwriting fields set in the original request.  Avoid doing the latter.
* This is a beta feature.  Future versions of Kubernetes may restrict the types of mutating webhooks can make.
* Future changes to control loops for built-in resources or third-party resources may break webhooks that work well today.  Even when the webhook installation API is field, not all possible webhook behaviors will be guaranteed to be supported indefinitely.

Reply:
Some tweaks to @erictune's additions:
```
Use caution when authoring and installing mutating webhooks.
* This is a beta feature. Future versions of Kubernetes may restrict the types of mutations webhooks can make.
* Users may be confused when the objects they try to create are different from what they receive. 
* Avoid overwriting fields. Setting originally unset fields is less confusing than overwriting fields set in the request.  
* Built in control loops may break when the objects they try to create are different when read back.
* Future changes to control loops for built-in resources or third-party resources may break webhooks that work well today. Even when the webhook installation API is a field, not all possible webhook behaviors will be guaranteed to be supported indefinitely.
```
So:
> Use caution when authoring and installing mutating webhooks.
> * Users may be confused when the objects they try to create are different from what they receive. 
> * Avoid overwriting fields. Setting originally unset fields is less confusing than overwriting fields set in the request.  
> * Built in control loops may break when the objects they try to create are different when read back.
> * This is a beta feature. Future versions of Kubernetes may restrict the types of mutations webhooks can make.
> * Future changes to control loops for built-in resources or third-party resources may break webhooks that work well today. Even when the webhook installation API is a field, not all possible webhook behaviors will be guaranteed to be supported indefinitely.